templates/README.md

```diff
@@ -0,0 +1,28 @@
+# Using templates to implement Secretless Connector Plugins
+
+We created connector templates to ease the process of adding new connectors to secretless.
+Before using the templates to add new connector plugins, be sure to read the [Secretless Connector Plugins README](https://github.com/cyberark/secretless-broker/blob/master/pkg/secretless/plugin/connector/README.md)
+
+To add a new connector do the following:
+
+1. Copy the relevant template directory (HTTP/TCP) into `internal/plugin/connectors/<protocol>`. 
```

Comment:
is `protocol` the language we're using for http vs tcp? it would be good to use consistent language to refer to this - I think I've seen it elsewhere as `connector type`

Reply:
i didn't find something consistent. I see "handler type", "connection type", "config type"
I guess "connector type" makes sense

Reply:
`connector type` 👍